### PR TITLE
[test_0] HelloWorld pr.

### PR DESCRIPTION
**Pull request by Kiameow**
### What?
- brief: 提交了队伍信息和个人信息以及helloWorld测试程序
- author: zhang_lumi@foxmail.com

### What changes are included in this PR?
添加了-队伍信息和个人信息以及helloWorld测试程序

### Are these changes tested?
截图 2023-06-16 12-24-52

### CODE REVIEW

File: stl/kia_meow/kiameow/src/helloWorld
Comment (Light-City) [inline stl/kia_meow/kiameow/src/helloWorld:1]: this file should put into .gitignore.

File: stl/kia_meow/kiameow/src/helloWorld.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include<iostream>
+
+int main() {
+	std::cout << "Hello World!" << std::endl;
```
Comment (Light-City) [inline stl/kia_meow/kiameow/src/helloWorld.cpp:4]: indent is not correct.

Comment (Kiameow) [reply stl/kia_meow/kiameow/src/helloWorld.cpp]: solved it, please check it again, thanks

**Comment by Light-City** [comment]
LGTM

**Pull request closed without merging**